app/Filters/UserFilter.php

```diff
@@ -38,6 +38,11 @@ public function allNames($name = '')
                            ->orWhere('last_name', 'like', "%$name%");
     }
 
+    public function role($role = '')
```

Comment:
There is already a "roles" query below, you need to pass an array like motion statuses